project/SparkBuild.scala

```diff
@@ -307,7 +307,9 @@ object SparkBuild extends PomBuild {
       DefaultMavenRepository,
       Resolver.mavenLocal,
       Resolver.file("ivyLocal", file(Path.userHome.absolutePath + "/.ivy2/local"))(Resolver.ivyStylePatterns)
-    ),
+    ) ++ Seq(
+      "staging-releases-mirror" at "https://repository.apache.org/content/repositories/staging/",
```

Comment:
Why is this required? Looks like a temporary change to me (to test out the change before parquet is widely available in the public repo).